infra/views.py

```diff
@@ -0,0 +1,17 @@
+'''Provide API views for infra module.'''
+from rest_framework import viewsets
+
+import infra.models
+import infra.serializers
+
+
+class OperationLogViewSet(viewsets.ModelViewSet):
+    '''Create API views for OperatinoLog'''
+    queryset = infra.models.OperationLog.objects.all()
+    serializer_class = infra.serializers.OperationLogSerializer
+
+
+class NotificationViewSet(viewsets.ModelViewSet):
```

Comment:
For Notification, It's created by System and we do not support update and delete, besides, users could only retrieve these notifications, so we should not allow HTTP methods other than GET.